models/User/UserRole.php

```diff
@@ -187,7 +187,7 @@ public function generatePermissionList()
     }
 
     /**
-     * @param string|array $permissions
+     * @param string|array|null $permissions
```

Comment:
why should we allow `null` here?

Reply:
as if it is null, it won't change anthing which might be not expected behavior...